test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -461,41 +472,75 @@ using INI = IniTestWrapper;
  * Type = Activation
  * Activation = softmax
  */
+// clang-format off
 INI fc_sigmoid_mse(
   "fc_sigmoid_mse",
-  {nn_base + "learning_rate=1 | optimizer=sgd | loss=mse | batch_size = 3",
-   I("input") + input_base + "input_shape = 1:1:3",
-   I("dense") + fc_base + "unit = 5", I("act") + sigmoid,
-   I("dense_1") + fc_base + "unit = 10", I("act_1") + softmax});
+  {
+    nn_base + "learning_rate=1 | optimizer=sgd | loss=mse | batch_size = 3",
```

Comment:
minor suggestion, randomize batch size in the test cases below where batch size `== 1` and `> 1` both happens?

Reply:
Added a new tc with one batch